packages/components/src/styles/components/form/super-select.scss

```diff
@@ -0,0 +1,502 @@
+      cursor: pointer;
+      content: "";
+    }
+
+    // unchecked checkbox:
+    &:not([aria-selected="true"]) {
+      &::before {
+        background-color: var(--token-form-control-base-surface-color-default);
+        border-color: var(--token-form-control-base-border-color-default);
+        box-shadow: var(--token-elevation-inset-box-shadow);
+      }
+    }
+
+    // unchecked checkbox, hovered:
+    &[aria-current="true"],
+    &:hover:not([aria-selected="true"]) {
+      &::before {
+        background-color: var(--token-form-control-base-surface-color-hover);
+        border-color: var(--token-form-control-base-border-color-hover);
+      }
+    }
+
+    // unchecked checkbox, disabled:
+    &[aria-disabled="true"]:not([aria-selected="true"]) {
+      &::before {
+        background-color: var(--token-form-control-disabled-surface-color);
+        border-color: var(--token-form-control-disabled-border-color);
+        box-shadow: none;
+        cursor: not-allowed;
+      }
+    }
+
+    // checked checkbox:
+    &[aria-selected="true"] {
+      &::before {
+        background-color: var(--token-form-control-checked-surface-color-default);
+        background-image: var(--token-form-checkbox-background-image-data-url);
+        border-color: var(--token-form-control-checked-border-color-default);
+      }
+
+      // checked checkbox, hovered:
+      &[aria-current="true"],
+      &:hover[aria-selected="true"] {
+        &::before {
+          background-color: var(--token-form-control-checked-border-color-default);
+          border-color: var(--token-form-control-checked-border-color-hover);
+        }
+      }
+
+      // checked checkbox, disabled:
+      &[aria-disabled="true"] {
+        &::before {
+          background-color: var(--token-form-control-disabled-surface-color);
+          background-image: var(--token-form-checkbox-background-image-data-url-disabled);
+          border-color: var(--token-form-control-disabled-border-color);
+          box-shadow: none;
+          cursor: not-allowed;
+        }
+      }
+    }
+  }
+}
+
+// NO MATCHES MESSAGE
+
+.hds-form-super-select .ember-basic-dropdown {
+  .ember-power-select-option--no-matches-message {
+    padding-left: 12px;
+    cursor: inherit;
+    // hide the hover indicator on messages
+    &::after {
+      display: none;
+    }
+  }
+}
+
+// AFTER OPTIONS
+
+.hds-form-super-select__after-options {
+  display: flex;
+  flex-wrap: wrap;
+  gap: 8px;
+  align-items: center;
+  padding: 8px 12px;
+  background-color: var(--token-color-surface-strong);
+  border-top: var(--token-form-control-border-width) solid var(--token-form-control-base-border-color-default);
+}
+
+.hds-form-super-select__result-count {
+  margin-left: auto;
+}
+
+.hds-form-super-select__no-options-selected {
+  min-height: $hds-super-select-item-height;
+  padding: 4px 15px 10px;
+  font-size: var(--token-typography-body-200-font-size);
```

Comment:
```suggestion
  font-size: var(--token-typography-body-200-font-size);
  font-family: var(--token-typography-body-200-font-family);
```

Reply:
The element with this class is now generated by `<Hds::Text:Body>` so instead of adding a `font-family` we'd likely need to remove the `font-size` (and `line-height`)

Reply:
Removed them in [40d6980](https://github.com/hashicorp/design-system/pull/2022/commits/40d69801ce4b5f342a83a26e6c3547be0c2b9c48)